protostar/testing/test_config.py

```diff
@@ -89,6 +94,8 @@ class TestConfig:
         default_factory=dict
     )
 
+    fuzz_examples: List[ExampleData] = field(default_factory=list)
```

Comment:
```suggestion
    fuzz_examples: PythonData = field(default_factory=dict)
```

Reply:
I think we need a list here.

Reply:
ah yes, you're right. use `List[PythonData]`